torchtune/training/memory.py

```diff
@@ -45,11 +45,11 @@ def set_activation_checkpointing(
 
 def cleanup_before_training() -> None:
     """
-    Call gc collect, empty CUDA cache, and reset peak memory stats.
+    Call gc collect, empty CUDA-like cache, and reset peak memory stats.
```

Comment:
nit: I think "device" should be sufficient here
```suggestion
    Call gc collect, empty device cache, and reset peak memory stats.
```

Reply:
Thank you for the suggestion! I’ll make the descriptions more appropriate.